### PR TITLE
Update Dell.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782, #10829

### CODE REVIEW

File: src/chrome/content/rules/Dell.xml [hunk 1]
```diff
@@ -109,7 +118,6 @@
 	<target host="ecomm.dell.com" />
 	<target host="ecomm2.dell.com" />
 	<target host="support.euro.dell.com" />
-	<target host="fcs.dell.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Dell.xml]: `http://fcs.dell.com/fcs/fcs/UIFramework/3.2.108.2/res/Image?path=sharedcontrols%2Fcontainers%2Fvalidation_messaging_left.png`

File: src/chrome/content/rules/Dell.xml [hunk 2]
```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://accessories.dell.com/ => https://accessories.dell.com/: Too many redirects while fetching 'https://accessories.dell.com/'
+Non-2xx HTTP code: http://fcs.dell.com/ (200) => https://fcs.dell.com/ (403)
+Fetch error: http://accessories-cdn.dell.com/ => https://accessories.dell.com/: Too many redirects while fetching 'https://accessories.dell.com/'
```
Comment (J0WI) [inline src/chrome/content/rules/Dell.xml]: Those comments can be removed.

File: src/chrome/content/rules/Dell.xml [hunk 3]
```diff
@@ -189,11 +198,16 @@
 	<rule from="^http://(?:www-cdn\.)?dell\.com/"
 		to="https://www.dell.com/" />
 
+	<!--	s? for protocol-relative
+		embeds on accessories:
+					-->
+	<rule from="^https?://accessories-cdn\.dell\.com/"
+		to="https://accessories.dell.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Dell.xml]: I think this is obsolete, isn't it?

**Comment by koops76** [comment]
@J0WI All is done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**